src/components/Details.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable prefer-destructuring */
```

Comment:
- [ ]  Kindly correct your linters instead of disabling the. they help you prevent future bugs in the app and help you apply best practices.